src/main/kotlin/racingCar/view/SplitString.kt

```diff
@@ -0,0 +1,19 @@
+package racingCar.view
+
+import java.lang.IllegalArgumentException
+
+object SplitString {
+    private const val DELIMITER = ","
+
+    fun getSplitNames(NamesOfCar: String): List<String> {
```

Comment:
변수명은 lowerCamelCase로!
```suggestion
    fun getSplitNames(namesOfCar: String): List<String> {
```